### PR TITLE
Disabled local store availability for showroom products.

### PR DESCRIPTION
**Pull request by Mauricio-Urrego**
[Remove "local availability" column on store stock information for specific categories](https://app.asana.com/0/30156186242982/1200685514567384/f)

### CODE REVIEW

File: src/Product.php [hunk 1]
```diff
@@ -207,6 +207,10 @@ public static function woocommerce_available_variation(array $variation_data, \W
    * Displays the shop stock-status component on the front-end.
    */
   public static function woocommerce_product_meta_start() {
+    if (get_the_terms(wc_get_product()->get_id(), 'product_cat')[0]->name === 'Ausstellungsstücke') {
```
Comment (sun) [inline src/Product.php:210]: 1. The `global $product` is already used below. You can also use it.
2. WordPress has a dedicated API function to test whether a post has a term; see https://rudrastyh.com/woocommerce/if-product-in-category.html
3. This is a shared plugin. The category name is a magic value. We do not need to make it configurable for now, but can you move the value into a class constant `Product::CATEGORY_EXCLUDED`?

**Review by sun — CHANGES_REQUESTED** [review]

File: src/Plugin.php [hunk 2]
```diff
@@ -53,9 +53,10 @@ public static function init() {
    * @implements wp_enqueue_scripts
    */
   public static function enqueueAssets() {
-    if (!is_product()) {
+    if (!is_product() || has_term(Product::CATEGORY_EXCLUDED, 'product_cat')) {
```
Comment (sun) [inline src/Plugin.php:56]: The logic of the condition is not right. The second condition should not be tested if the first is already negative.

**Review by sun — CHANGES_REQUESTED** [review]

File: src/Plugin.php (hunk 2 above)
Comment (fabianmarz) [reply src/Plugin.php]: @Mauricio-Urrego, in my opinion this condition is correct. 🤔 PHP will not evaluate the second condition if don't have to. So if it's not a product, it won't run the `has_term` condition. I assume this was working correctly before, or?

File: src/Plugin.php [hunk 3]
```diff
@@ -56,6 +56,11 @@ public static function enqueueAssets() {
     if (!is_product()) {
       return;
     }
+
+    if (has_term(Product::CATEGORY_EXCLUDED, 'product_cat')) {
```
Comment (fabianmarz) [inline src/Plugin.php:60]: Please see my comment in the conversation. I think your previous code was totally fine 👍

**Review by fabianmarz — APPROVED** [review]
LGTM :)

**Pull request merged**